Dockerfile.tmpl

```diff
@@ -81,7 +89,8 @@ RUN conda config --add channels nvidia && \
 
 # b/232247930: uninstall pyarrow to avoid double installation with the GPU specific version.
 RUN pip uninstall -y pyarrow && \
-    conda install cudf=21.10 cuml=21.10 cudatoolkit=$CUDA_MAJOR_VERSION.$CUDA_MINOR_VERSION && \
+    conda install -c conda-forge mamba && \
+    mamba install -y cudf cuml cudatoolkit==11.2.2 && \
```

Comment:
#15 334.5 Looking for: ['cudf', 'cuml', 'cudatoolkit==11.2.2']

#15 334.5 

#15 334.5 

#15 334.5 Pinned packages:

#15 334.5   - python 3.7.*

#15 334.5 

#15 334.5 

#15 334.5 Encountered problems while solving:

#15 334.5   - nothing provides cuda92 needed by libcuml-0.5.0-cuda9.2_3